### DISCUSSION TITLE
Does Bullseye work?

### DISCUSSION THREAD

**Discussion by sanrobin** [dnschneid/crouton#4572] [Q&A] [answered]
Hello There,

As Yesterday, Debian 11 Bullseye was released, I wanted to install it. But when I boot up, It crashes. I will link a screenshot.DOES IT EVEN WORK?
I even tried reinstalling but to no avail.

[Screenshot](https://drive.google.com/file/d/19sQggjQsVXctkYtSC4eG2Dp8E_iC_UVx/view?usp=sharing)

**Comment by ghost** [comment, marked as answer]
This seems to be an issue with early releases of distros, as @DennisLfromGA said, try it in an earlier release ([here](https://github.com/dnschneid/crouton/issues/4368#issuecomment-719007419)).

**Reply by sanrobin** [reply]
It may be the problem after all🤔. I ended up using Buster though. Since Bullseye released this week, it may have to get fixed.
